### ISSUE TITLE
Deprecate notice

### ISSUE THREAD

**Issue by mtripg6666tdr** [mtripg6666tdr/djs-command-resolver#1]
# TL;DR
This pacakge is marked as deprecated, which means this pacakge is no longer supported and will not receive any updates.
Please build your own system, or fork this repository to keep up-to-date yourself.

# Why?
This package was originally splitted from my own bot as mentioned in README. These days discord.js officially stated that they don't support TypeScript environments without setting `strict` to true [here](https://github.com/discordjs/discord.js/issues/7924).
I don't like the policy and I like coding with `strict: false`.
So I decided to stop using discord.js someday and replace it with [eris](https://github.com/abalabahaha/eris), another api wrapper in js.
For more information about my decision, you can refer [this issue (Japanese)](https://github.com/mtripg6666tdr/Discord-SimpleMusicBot/issues/256) in my bot repository.
I will make something like command resolver for eris in future. If you are an eris user or an user being in progress migrating from discord.js to eris, it might be useful.

# What we should do?
You can use this package until occurring an error, still without any supports or updates.
You have to rewrite your bot without this, or you have to maintain this package yourself for you and everyone using discord.js.

Finally I wanna say thanks for everyone finding and using this package.
